README.md

```diff
@@ -53,19 +51,21 @@ Use in conjunction with lint and spec tests for Continuous Integration:
 
 ## Checks
 
-### hiera
+[TODO: this lists all the checks this makes, but where does it check? In the directory that the Rakefile is in, or...?]
+
+### Hiera
 
 Checks `.yaml` files for syntax errors.
 
-By default, the rake task will look for all `.yaml` files under:
+By default, the rake task looks for all `.yaml` files under:
 
 * `**/data/**/*.yaml`
 * `hieradata/**/*.yaml`
 * `hiera*.yaml`
```

Comment:
Note: these patterns would only work for the single-module usage. A control-repo usage would need to replace those in configuration.